make/autoconf/toolchain.m4

```diff
@@ -1000,6 +935,13 @@ AC_DEFUN_ONCE([TOOLCHAIN_MISC_CHECKS],
       fi
     fi
   fi
+  if test "x$OPENJDK_TARGET_OS" = xaix; then
+    # Make sure we have the Open XL version of clang on AIX
+    $ECHO "$CC_VERSION_OUTPUT" | $GREP "IBM Open XL C/C++ for AIX" > /dev/null
```

Comment:
This does not work since $CC_VERSION_OUTPUT is unset. We need 
	CC_VERSION_OUTPUT=`${XLC_TEST_PATH}ibm-clang++_r --version 2>&1 | $HEAD -n 1`
before, as in the previous code some lines above which you removed.

Reply:
Yeah, you are right. I confused it with CC_VERSION_STRING.

... duh.

Actually, that is what I should be using instead.